ebean-test/src/test/java/misc/migration/v1_1/DropMain.java

```diff
@@ -1,4 +1,4 @@
-package misc.migration.v1_0;
+package misc.migration.v1_1;
```

Comment:
The classes form an earlier test exists only in `v1_0`
As the directories `v1_0` and `v1_2` should be equal, I decided to move them to `v1_1` instead of copying them